src/genie/libs/parser/iosxr/show_bgp.py

```diff
@@ -5078,12 +5079,13 @@ def cli(self, vrf_type='all', address_family='', instance='all', vrf='all', outp
         # *>i10.9.2.0/24        10.64.4.4               2219    100      0 400 33299 51178 47751 {27016} e
         # *>i10.169.1.0/24      10.64.4.4               2219    100      0 300 33299 51178 47751 {27016} e
         # *>i192.168.111.0/24       10.189.99.98                                                    0       0 i
+        # *> 10.7.7.7/32        10.10.10.107             0             0 65107.65107 ?
         p16 = re.compile(r'^(?P<status_codes>(i|s|x|S|d|h|\*|\>|\s)+)'
                          r' *(?P<prefix>(?P<ip>[0-9\.\:\[\]]+)/(?P<mask>\d+))?'
-                         r' +(?P<next_hop>\S+) +(?P<number>[\d\s\{\}]+)'
+                         r' +(?P<next_hop>\S+) +(?P<number>[\d\.\s\{\}]+)'
                          r'(?: *(?P<origin_codes>(i|e|\?)))?$')
 
-        p17 = re.compile(r'(?P<path>[\d\s]+)'
+        p17 = re.compile(r'(?P<path>[\d\.\s]+)'
```

Comment:
Please add a comment with an example

Reply:
I modified all RegEx used for capturing ASN in the path. And I am not able to replicate some of the output. But I put the possible example.